pkg/modules/servicesqb/dto.go

```diff
@@ -0,0 +1,44 @@
+package servicesqb
+
+import "time"
+
+// Request mirrors the /services API input shape
+type Request struct {
+	Start uint64          `json:"start"`
+	End   uint64          `json:"end"`
+	Tags  []TagFilterItem `json:"tags"`
+}
```

Comment:
```suggestion
type Request struct {
	Start uint64          `json:"start"`
	End   uint64          `json:"end"`
	Tags  []TagFilterItem `json:"tags"`
}
```

Please move to types layer

Reply:
done